LivingSpace/Localization/XComGame.int

```diff
@@ -7,3 +7,9 @@ Summary="Increases crew capacity by 10, allowing more personnel to be maintained
 DisplayName="Crew Size II"
 FacilityName="Living Quarters"
 Summary="Increases crew capacity by a further 10, allowing even more personnel to be maintained without an increase to recovery times."
+
+[Laboratory X2FacilityTemplate]
+Summary="The Laboratory prevents Scientists from counting towards the Crew Limit, and increases the chance that a research breakthrough will occur. Scientists can be staffed in laboratories to provide additional assistance to Dr. Tygan, further decreasing the time of all research projects."
+
+[Workshop X2FacilityTemplate]
+Summary="The Workshop prevents Engineers from counting towards the Crew Limit. Engineers can also be staffed in workshops, creating unique GREMLIN drones which can be sent to staff any adjacent room or facility."
```

Comment:
This will conflict with any mods that change these facilities, whereas the code won't (as long as the template names are the same). Maybe there any other places/ways of conveying this info?

Reply:
We can make a note in the CI facilities tutorial perhaps, the problem is that it only appears once at the start of the game

Reply:
Now this part is especially bad, considering it doesn't respect the config.

I think the popup is enough for LS and for CI the tutorial + https://github.com/WOTCStrategyOverhaul/CovertInfiltration/issues/582 should also be enough